.github/workflows/amd-offline-tests.yml

```diff
@@ -27,29 +27,22 @@ jobs:
         run: |
           echo "PATH=${HOME}/.local/bin:${PATH}" >> "${GITHUB_ENV}"
 
-      # - name: Check pre-commit
-      #   if: ${{ matrix.runner != 'macos-10.15' }}
-      #   run: |
-      #     apt remove -y cmake
-      #     apt update
-      #     apt install -y git
-      #     # python3 -m pip install --upgrade pre-commit
-      #     # python3 -m pre_commit run --all-files
-
       - name: Prerequisite
         run: |
           # remove system cmake to use python cmake instead
-          apt remove -y cmake
-          pip3 install cmake
           apt update
-          apt install -y libpython3.8-dev
+          apt install -y libpython3-dev python3-pip git wget
+          # get rocm related utilities
+          wget https://repo.radeon.com/amdgpu-install/5.3/ubuntu/jammy/amdgpu-install_5.3.50300-1_all.deb
+          apt-get install -y ./amdgpu-install_5.3.50300-1_all.deb
+          amdgpu-install -y --usecase=rocm --no-dkms
+
+          pip3 install cmake
+          pip3 install torch==1.13.1 --index-url https://download.pytorch.org/whl/rocm5.2
```

Comment:
And torch for ROCM5.5

Reply:
torch with rocm 5.5 is not available from this source yet. 
not sure where can I get it :slightly_frowning_face:, so I used 5.4.2

I have some concerns about new pytorch and rocm sizes, btw. We will see soon if it fits.
If it is not, I want ask CI team to add jenkins job for offline tests.